engines/osxnative/config-schema.yml

```diff
@@ -0,0 +1,37 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              Config
+description: |-
+  Config applicable to Mac OSX native engine
+type:               object
+properties:
+  workerGroup:
+    title:          Worker Group
```

Comment:
So, we end up with an empty schema for now? cc @petemoore as this also applies for winnative engine.